src/connector.js

```diff
@@ -863,7 +863,8 @@ function updateResultListState( newState ) {
 				.replace( '%[short-date-fr]', getShortDateFormat( resultDate ) )
 				.replace( '%[long-date-en]', getLongDateFormat( resultDate, 'en' ) )
 				.replace( '%[long-date-fr]', getLongDateFormat( resultDate, 'fr' ) )
-				.replace( '%[highlightedExcerpt]', highlightedExcerpt );
+				.replace( '%[highlightedExcerpt]', highlightedExcerpt )
+				.replace( /&/g, "&amp;" );
```

Comment:
@Basselbi What happens if there is an already escaped string in the input HTML, for example `&amp;`, wouldn't this convert it to `&amp;amp;` in the output?